index.md

```diff
@@ -2,24 +2,76 @@
 
 Welcome to PyTA - Your guide to code in a standard way.
 
-## Pylint errors {#pylint}
+## Improper Python usage
```

Comment:
This is true - pandoc actually generates ids automatically for all headers, based on their text. You use `{#...}` to provide your own custom id instead.